src/main/java/uk/gov/justice/digital/delius/service/CaseloadService.java

```diff
@@ -38,11 +39,11 @@ public Optional<Caseload> getCaseloadByStaffIdentifier(final long staffIdentifie
     public Optional<Caseload> getCaseloadByTeamCode(
         final String teamCode, final Pageable pageable, final CaseloadRole... roles
     ) {
-        if (!teamRepository.existsByCode(teamCode)) return Optional.empty();
```

Comment:
You might also be able to remove the existsByCode method from the interface now.